wp-content/themes/workingnyc/timber-posts/Jobs.php

```diff
@@ -180,8 +180,7 @@ public function getOrganization() {
    * @return  String  Summary with HTML markup stripped
    */
   public function getSummary($limit = 120) {
```

Comment:
Where is this function used? Is it okay to remove it if it just returns an empty string?

Reply:
Yes we can remove it but it was calling from 2 places line number 81 and 119. If we need to remove then also we need to remove these 2 calls along with changes in job.twig.

Reply:
Yes, you should remove those lines. And you should also check where `summary` from lines 81 and 119 are being used and remove those as well.